### PR TITLE
Feature/include tags

### PR DESCRIPTION
**Pull request by MikeLundahl**
Include tags in the feed as an option. 

image

screen from TT RSS
image

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "blomstra/syndication",
+    "name": "ianm/syndication",
```
Comment (MikeLundahl) [inline composer.json]: I had to change this in order to get it installed on my dev instance. Should probably change this back?

Comment (luceos) [reply composer.json]: Yeah definitely needs to be reverted.

**Comment by MikeLundahl** [comment]
I discovered some small things. I have pushed a fix to check for undefined variable in the blade template. Also, I made sure that tags are now included when subscribing to a single thread/discussion.

File: composer.json (hunk 1 above)
Comment (MikeLundahl) [reply composer.json]: I pushed the change back to blomstra

**Pull request merged**